app/src/main/java/com/eveningoutpost/dexdrip/Services/LibreWifiReader.java

```diff
@@ -217,6 +221,81 @@ private static boolean hasGeoLocation(LibreWifiData record) {
         
     }
 
+    // read from http source like cloud hosted parakeet receiver.cgi / json.get
+    private static List<LibreWifiData> readHttpJson(String url, int numberOfRecords) {
+        long newest_timestamp = 0;
+        final long time_start = JoH.tsl();
+        final List<LibreWifiData> trd_list = new LinkedList<LibreWifiData>();
+        try {
+
+            if (httpClient == null) {
+                httpClient = new OkHttpClient();
+                // suitable for GPRS
+                httpClient.setConnectTimeout(30, TimeUnit.SECONDS);
+                httpClient.setReadTimeout(60, TimeUnit.SECONDS);
+                httpClient.setWriteTimeout(20, TimeUnit.SECONDS);
+            }
+
+
+            // simple HTTP GET request
+            // n=numberOfRecords for backfilling
+            // r=sequence number to avoid any cache
+            // expecting json reply like the standard json server in dexterity / python pi usb / parakeet
+            final Request request = new Request.Builder()
+                    // Mozilla header facilitates compression
+                    .header("User-Agent", "Mozilla/5.0")
+                    .header("Connection", "close")
+                    //.url(url + "?n=" + Integer.toString(numberOfRecords)
```

Comment:
Please remove it here or move it to the JavaDoc of this function.

Reply:
Done.

Reply:
Done.